salt/states/zabbix_host.py

```diff
@@ -122,8 +122,8 @@ def _interface_format(interfaces_data):
             interface_type = interface_ports[value['type'].lower()][0]
             main = '1' if six.text_type(value.get('main', 'true')).lower() == 'true' else '0'
             useip = '1' if six.text_type(value.get('useip', 'true')).lower() == 'true' else '0'
-            interface_ip = value.get('ip')
-            dns = value.get('dns', key)
+            interface_ip = value.get('ip') or ""
```

Comment:
We prefer single quotes in Salt. Also, we should probably use the optional `value` option.
```
interface_ip = value.get('ip', '')
```

Reply:
You're right, that's much cleaner. Changed.